### PR TITLE
Add bw2parameters

### PR DESCRIPTION
**Pull request by cmutel**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cmutel** [comment]
I agree to maintain

**Comment by cmutel** [comment]
@conda-forge/help-python, this is ready to review. Thanks for your time and support.

File: recipes/bw2parameters/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/brightway-lca/brightway2-parameters
+  license: BSD-2-Clause
+  license_file: LICENSE
+  summary: 'Library for storing, validating, and calculating with parameters. Designed to work with the Brightway2 life cycle assessment framework, but is generic enough to work in other use cases.'
```
Comment (chrisburr) [inline recipes/bw2parameters/meta.yaml:42]: The summary should be short:
```suggestion
  summary: Library for storing, validating, and calculating with parameters
  description: |
      Designed to work with the Brightway2 life cycle assessment framework, but is
      generic enough to work in other use cases.
```

File: recipes/bw2parameters/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  license_family: BSD
```
Comment (chrisburr) [inline recipes/bw2parameters/meta.yaml:43]: These isn't needed these days:
```suggestion
```

File: recipes/bw2parameters/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+{% set version = "0.7" %}
+{% set sha256 = "f9d6cd45488762a755ede5c11754d5acf790f1e8ab592a86babd02f91c08a87a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (chrisburr) [inline recipes/bw2parameters/meta.yaml:11]:
```suggestion
  sha256: f9d6cd45488762a755ede5c11754d5acf790f1e8ab592a86babd02f91c08a87a
```

File: recipes/bw2parameters/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "bw2parameters" %}
+{% set version = "0.7" %}
+{% set sha256 = "f9d6cd45488762a755ede5c11754d5acf790f1e8ab592a86babd02f91c08a87a" %}
```
Comment (chrisburr) [inline recipes/bw2parameters/meta.yaml:3]: As it's only used once it's better to not bother with a variable for `sha256`:
```suggestion
```

**Pull request merged**